lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -382,12 +356,9 @@ def model_create_prm_any_baseline_building(user_model, building_type, climate_zo
         air_loop_hvac_apply_minimum_vav_damper_positions(air_loop, false)
       end
 
-      if /prm/i !~ template
-        # If there are any multi-zone systems, reset damper positions to achieve a 60% ventilation effectiveness minimum for the system
-        # following the ventilation rate procedure from 62.1
-        model_apply_multizone_vav_outdoor_air_sizing(model)
-
-      end
+      # If there are any multi-zone systems, reset damper positions to achieve a 60% ventilation effectiveness minimum for the system
+      # following the ventilation rate procedure from 62.1
+      model_apply_multizone_vav_outdoor_air_sizing(model)
```

Comment:
Do we actually want this to occur for the 90.1-2019 PRM? I thought we did not find it a requirement for it or a mention of it in the PRM-RM.

Reply:
Right. The 90_1_prm version just returns "true".